From 14a10071fbd44c9cefb881884023861300105db0 Mon Sep 17 00:00:00 2001
From: Etienne Carriere <etienne.carriere@foss.st.com>
Date: Mon, 16 Oct 2023 17:32:04 +0200
Subject: [PATCH 0010/1695] core: prefix header file guard names with __

Improves header files guard names consistency by using a __ prefix
where missing.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git bcd234c874fdab1553dfc20dcea4ad7c1c8887b8 core: prefix header file guard names with __]
Change-Id: Ie33c0a89093da7e5a9dcd8c3f51cabb0a16d30bd
Signed-off-by: Etienne Carriere <etienne.carriere@foss.st.com>
[etienne: cherry-picked from d50fee0321fe6853ac6352cf0fd548666457b407]
Signed-off-by: Etienne Carriere <etienne.carriere@foss.st.com>
---
 core/arch/arm/include/arm.h                | 6 +++---
 core/arch/arm/include/arm32.h              | 6 +++---
 core/arch/arm/include/arm64.h              | 6 +++---
 core/arch/arm/include/kernel/misc_arch.h   | 6 +++---
 core/arch/arm/include/kernel/vfp.h         | 6 +++---
 core/arch/arm/include/sm/pm.h              | 4 ++--
 core/arch/arm/include/sm/sm.h              | 6 +++---
 core/arch/arm/include/sm/watchdog_smc.h    | 6 +++---
 core/arch/arm/include/tee/entry_fast.h     | 6 +++---
 core/arch/riscv/include/kernel/misc_arch.h | 6 +++---
 core/arch/riscv/include/riscv.h            | 6 +++---
 core/arch/riscv/include/sbi.h              | 6 +++---
 core/arch/riscv/include/tee/entry_fast.h   | 6 +++---
 core/include/bench.h                       | 6 +++---
 core/include/console.h                     | 6 +++---
 core/include/crypto/sm2-kdf.h              | 6 +++---
 core/include/drivers/cbmem_console.h       | 6 +++---
 core/include/drivers/gpio.h                | 6 +++---
 core/include/drivers/plic.h                | 6 +++---
 core/include/drivers/regulator.h           | 6 +++---
 core/include/drivers/rtc.h                 | 6 +++---
 core/include/drivers/stm32_gpio.h          | 6 +++---
 core/include/drivers/stm32_i2c.h           | 6 +++---
 core/include/drivers/stm32_iwdg.h          | 6 +++---
 core/include/drivers/stm32_uart.h          | 6 +++---
 core/include/drivers/wdt.h                 | 6 +++---
 core/include/gen-asm-defines.h             | 6 +++---
 core/include/initcall.h                    | 4 ++--
 core/include/io.h                          | 6 +++---
 core/include/keep.h                        | 6 +++---
 core/include/kernel/abort.h                | 6 +++---
 core/include/kernel/dt.h                   | 6 +++---
 core/include/kernel/early_ta.h             | 6 +++---
 core/include/kernel/embedded_ts.h          | 6 +++---
 core/include/kernel/handle.h               | 6 +++---
 core/include/kernel/ldelf_loader.h         | 6 +++---
 core/include/kernel/ldelf_syscalls.h       | 6 +++---
 core/include/kernel/misc.h                 | 6 +++---
 core/include/kernel/msg_param.h            | 6 +++---
 core/include/kernel/mutex.h                | 6 +++---
 core/include/kernel/panic.h                | 6 +++---
 core/include/kernel/pseudo_ta.h            | 6 +++---
 core/include/kernel/spinlock.h             | 6 +++---
 core/include/kernel/thread.h               | 6 +++---
 core/include/kernel/unwind.h               | 6 +++---
 core/include/kernel/user_ta.h              | 6 +++---
 core/include/kernel/virtualization.h       | 6 +++---
 core/include/kernel/wait_queue.h           | 6 +++---
 core/include/mm/pgt_cache.h                | 6 +++---
 core/include/mm/tee_pager.h                | 4 ++--
 core/include/optee_msg.h                   | 6 +++---
 core/include/signed_hdr.h                  | 6 +++---
 core/include/tee/cache.h                   | 6 +++---
 core/include/tee/entry_std.h               | 6 +++---
 54 files changed, 159 insertions(+), 159 deletions(-)

diff --git a/core/arch/arm/include/arm.h b/core/arch/arm/include/arm.h
index 0eea43856..43504ba5b 100644
--- a/core/arch/arm/include/arm.h
+++ b/core/arch/arm/include/arm.h
@@ -3,8 +3,8 @@
  * Copyright (c) 2015, Linaro Limited
  * Copyright (c) 2019-2023, Arm Limited. All rights reserved.
  */
-#ifndef ARM_H
-#define ARM_H
+#ifndef __ARM_H
+#define __ARM_H
 
 #include <stdbool.h>
 #include <stdint.h>
@@ -200,4 +200,4 @@ static inline bool feat_pauth_is_implemented(void)
 
 #endif
 
-#endif /*ARM_H*/
+#endif /*__ARM_H*/
diff --git a/core/arch/arm/include/arm32.h b/core/arch/arm/include/arm32.h
index 253b197ff..ef729d492 100644
--- a/core/arch/arm/include/arm32.h
+++ b/core/arch/arm/include/arm32.h
@@ -4,8 +4,8 @@
  * Copyright (c) 2014, STMicroelectronics International N.V.
  */
 
-#ifndef ARM32_H
-#define ARM32_H
+#ifndef __ARM32_H
+#define __ARM32_H
 
 #include <compiler.h>
 #include <sys/cdefs.h>
@@ -278,4 +278,4 @@ static __always_inline __noprof uint32_t read_r7(void)
 
 #endif /*__ASSEMBLER__*/
 
-#endif /*ARM32_H*/
+#endif /*__ARM32_H*/
diff --git a/core/arch/arm/include/arm64.h b/core/arch/arm/include/arm64.h
index 8c210b3fa..829539473 100644
--- a/core/arch/arm/include/arm64.h
+++ b/core/arch/arm/include/arm64.h
@@ -3,8 +3,8 @@
  * Copyright (c) 2015, Linaro Limited
  * Copyright (c) 2023, Arm Limited
  */
-#ifndef ARM64_H
-#define ARM64_H
+#ifndef __ARM64_H
+#define __ARM64_H
 
 #include <compiler.h>
 #include <sys/cdefs.h>
@@ -504,5 +504,5 @@ static inline void write_pan_disable(void)
 
 #endif /*__ASSEMBLER__*/
 
-#endif /*ARM64_H*/
+#endif /*__ARM64_H*/
 
diff --git a/core/arch/arm/include/kernel/misc_arch.h b/core/arch/arm/include/kernel/misc_arch.h
index 0653119d5..992c75f7c 100644
--- a/core/arch/arm/include/kernel/misc_arch.h
+++ b/core/arch/arm/include/kernel/misc_arch.h
@@ -3,8 +3,8 @@
  * Copyright (c) 2014, STMicroelectronics International N.V.
  */
 
-#ifndef KERNEL_MISC_ARCH_H
-#define KERNEL_MISC_ARCH_H
+#ifndef __KERNEL_MISC_ARCH_H
+#define __KERNEL_MISC_ARCH_H
 
 #include <arm.h>
 #include <kernel/thread.h>
@@ -17,4 +17,4 @@ uint32_t read_mode_lr(int cpu_mode);
 
 void wait_cycles(unsigned long cycles);
 
-#endif /*KERNEL_MISC_ARCH_H*/
+#endif /*__KERNEL_MISC_ARCH_H*/
diff --git a/core/arch/arm/include/kernel/vfp.h b/core/arch/arm/include/kernel/vfp.h
index 49c0d736c..f42ca9795 100644
--- a/core/arch/arm/include/kernel/vfp.h
+++ b/core/arch/arm/include/kernel/vfp.h
@@ -3,8 +3,8 @@
  * Copyright (c) 2015, Linaro Limited
  */
 
-#ifndef KERNEL_VFP_H
-#define KERNEL_VFP_H
+#ifndef __KERNEL_VFP_H
+#define __KERNEL_VFP_H
 
 #include <types_ext.h>
 #include <compiler.h>
@@ -103,4 +103,4 @@ void vfp_lazy_save_state_final(struct vfp_state *state, bool force_save);
  */
 void vfp_lazy_restore_state(struct vfp_state *state, bool full_state);
 
-#endif /*KERNEL_VFP_H*/
+#endif /*__KERNEL_VFP_H*/
diff --git a/core/arch/arm/include/sm/pm.h b/core/arch/arm/include/sm/pm.h
index 939f966e8..351095a9b 100644
--- a/core/arch/arm/include/sm/pm.h
+++ b/core/arch/arm/include/sm/pm.h
@@ -26,8 +26,8 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
-#ifndef SM_PM_H
-#define SM_PM_H
+#ifndef __SM_PM_H
+#define __SM_PM_H
 #include <stdint.h>
 #include <types_ext.h>
 
diff --git a/core/arch/arm/include/sm/sm.h b/core/arch/arm/include/sm/sm.h
index d33c5fc5c..9e7b98791 100644
--- a/core/arch/arm/include/sm/sm.h
+++ b/core/arch/arm/include/sm/sm.h
@@ -4,8 +4,8 @@
  * Copyright (c) 2014, STMicroelectronics International N.V.
  */
 
-#ifndef SM_SM_H
-#define SM_SM_H
+#ifndef __SM_SM_H
+#define __SM_SM_H
 
 #ifndef __ASSEMBLER__
 
@@ -142,4 +142,4 @@ void vector_fiq_entry(uint32_t a0, uint32_t a1, uint32_t a2, uint32_t a3,
 #define SM_EXIT_TO_NON_SECURE		0
 #define SM_EXIT_TO_SECURE		1
 
-#endif /*SM_SM_H*/
+#endif /*__SM_SM_H*/
diff --git a/core/arch/arm/include/sm/watchdog_smc.h b/core/arch/arm/include/sm/watchdog_smc.h
index b8b55a489..555d519f7 100644
--- a/core/arch/arm/include/sm/watchdog_smc.h
+++ b/core/arch/arm/include/sm/watchdog_smc.h
@@ -2,8 +2,8 @@
 /*
  * Copyright (C) STMicroelectronics 2023
  */
-#ifndef SM_WATCHDOG_SMC_H
-#define SM_WATCHDOG_SMC_H
+#ifndef __SM_WATCHDOG_SMC_H
+#define __SM_WATCHDOG_SMC_H
 
 /*
  * This file describes the secure watchdog management service.
@@ -78,4 +78,4 @@
  * a7		Hypervisor Client ID register
  */
 
-#endif /* SM_WATCHDOG_SMC_H */
+#endif /* __SM_WATCHDOG_SMC_H */
diff --git a/core/arch/arm/include/tee/entry_fast.h b/core/arch/arm/include/tee/entry_fast.h
index 0f6fef8cc..b6e77dabf 100644
--- a/core/arch/arm/include/tee/entry_fast.h
+++ b/core/arch/arm/include/tee/entry_fast.h
@@ -4,8 +4,8 @@
  * Copyright (c) 2014, STMicroelectronics International N.V.
  */
 
-#ifndef TEE_ENTRY_FAST_H
-#define TEE_ENTRY_FAST_H
+#ifndef __TEE_ENTRY_FAST_H
+#define __TEE_ENTRY_FAST_H
 
 #include <kernel/thread.h>
 
@@ -30,4 +30,4 @@ size_t tee_entry_generic_get_api_call_count(void);
 void tee_entry_fast(struct thread_smc_args *args);
 void __tee_entry_fast(struct thread_smc_args *args);
 
-#endif /* TEE_ENTRY_FAST_H */
+#endif /* __TEE_ENTRY_FAST_H */
diff --git a/core/arch/riscv/include/kernel/misc_arch.h b/core/arch/riscv/include/kernel/misc_arch.h
index 208e358f5..c48c7d9a8 100644
--- a/core/arch/riscv/include/kernel/misc_arch.h
+++ b/core/arch/riscv/include/kernel/misc_arch.h
@@ -3,7 +3,7 @@
  * Copyright 2022 NXP
  */
 
-#ifndef KERNEL_MISC_ARCH_H
-#define KERNEL_MISC_ARCH_H
+#ifndef __KERNEL_MISC_ARCH_H
+#define __KERNEL_MISC_ARCH_H
 
-#endif /*KERNEL_MISC_ARCH_H*/
+#endif /*__KERNEL_MISC_ARCH_H*/
diff --git a/core/arch/riscv/include/riscv.h b/core/arch/riscv/include/riscv.h
index ecaba09b5..0b4c1ffa4 100644
--- a/core/arch/riscv/include/riscv.h
+++ b/core/arch/riscv/include/riscv.h
@@ -3,8 +3,8 @@
  * Copyright 2022-2023 NXP
  */
 
-#ifndef RISCV_H
-#define RISCV_H
+#ifndef __RISCV_H
+#define __RISCV_H
 
 #include <compiler.h>
 #include <encoding.h>
@@ -389,4 +389,4 @@ static inline __noprof uint32_t read_cntfrq(void)
 
 #endif /*__ASSEMBLER__*/
 
-#endif /*RISCV_H*/
+#endif /*__RISCV_H*/
diff --git a/core/arch/riscv/include/sbi.h b/core/arch/riscv/include/sbi.h
index 6ea6064d8..8b73edee9 100644
--- a/core/arch/riscv/include/sbi.h
+++ b/core/arch/riscv/include/sbi.h
@@ -3,8 +3,8 @@
  * Copyright 2022 NXP
  */
 
-#ifndef SBI_H
-#define SBI_H
+#ifndef __SBI_H
+#define __SBI_H
 
 #if defined(CFG_RISCV_SBI)
 
@@ -44,4 +44,4 @@ int sbi_boot_hart(uint32_t hart_id, paddr_t start_addr, unsigned long arg);
 
 #endif /*__ASSEMBLER__*/
 #endif /*defined(CFG_RISCV_SBI)*/
-#endif /*SBI_H*/
+#endif /*__SBI_H*/
diff --git a/core/arch/riscv/include/tee/entry_fast.h b/core/arch/riscv/include/tee/entry_fast.h
index 08ca181f4..b3f1f81b7 100644
--- a/core/arch/riscv/include/tee/entry_fast.h
+++ b/core/arch/riscv/include/tee/entry_fast.h
@@ -5,8 +5,8 @@
  * Copyright (c) 2014, STMicroelectronics International N.V.
  */
 
-#ifndef TEE_ENTRY_FAST_H
-#define TEE_ENTRY_FAST_H
+#ifndef __TEE_ENTRY_FAST_H
+#define __TEE_ENTRY_FAST_H
 
 #include <kernel/thread.h>
 
@@ -31,4 +31,4 @@ size_t tee_entry_generic_get_api_call_count(void);
 void tee_entry_fast(struct thread_abi_args *args);
 void __tee_entry_fast(struct thread_abi_args *args);
 
-#endif /* TEE_ENTRY_FAST_H */
+#endif /* __TEE_ENTRY_FAST_H */
diff --git a/core/include/bench.h b/core/include/bench.h
index cdff57734..d9f3cdd3f 100644
--- a/core/include/bench.h
+++ b/core/include/bench.h
@@ -3,8 +3,8 @@
  * Copyright (c) 2017, Linaro Limited
  */
 
-#ifndef BENCH_H
-#define BENCH_H
+#ifndef __BENCH_H
+#define __BENCH_H
 
 #include <inttypes.h>
 #include <mm/core_memprot.h>
@@ -57,4 +57,4 @@ void bm_timestamp(void);
 static inline void bm_timestamp(void) {}
 #endif /* CFG_TEE_BENCHMARK */
 
-#endif /* BENCH_H */
+#endif /* __BENCH_H */
diff --git a/core/include/console.h b/core/include/console.h
index f2e57fb7d..f9cfaca51 100644
--- a/core/include/console.h
+++ b/core/include/console.h
@@ -3,8 +3,8 @@
  * Copyright (c) 2014, Linaro Limited
  */
 
-#ifndef CONSOLE_H
-#define CONSOLE_H
+#ifndef __CONSOLE_H
+#define __CONSOLE_H
 
 #include <compiler.h>
 #include <tee_api_types.h>
@@ -51,5 +51,5 @@ static inline void configure_console_from_dt(void)
 {}
 #endif /* !CFG_DT */
 
-#endif /* CONSOLE_H */
+#endif /* __CONSOLE_H */
 
diff --git a/core/include/crypto/sm2-kdf.h b/core/include/crypto/sm2-kdf.h
index 5a012c0d9..4ac246bbc 100644
--- a/core/include/crypto/sm2-kdf.h
+++ b/core/include/crypto/sm2-kdf.h
@@ -3,11 +3,11 @@
  * Copyright (c) 2020-2021, Huawei Technologies Co., Ltd
  */
 
-#ifndef CRYPTO_SM2_KDF_H
-#define CRYPTO_SM2_KDF_H
+#ifndef __CRYPTO_SM2_KDF_H
+#define __CRYPTO_SM2_KDF_H
 
 #include <stdint.h>
 #include <tee_api_types.h>
 
 TEE_Result sm2_kdf(const uint8_t *Z, size_t Z_len, uint8_t *t, size_t tlen);
-#endif /* CRYPTO_SM2_KDF_H */
+#endif /* __CRYPTO_SM2_KDF_H */
diff --git a/core/include/drivers/cbmem_console.h b/core/include/drivers/cbmem_console.h
index c8234d3b6..a8b695a93 100644
--- a/core/include/drivers/cbmem_console.h
+++ b/core/include/drivers/cbmem_console.h
@@ -2,12 +2,12 @@
 /*
  * Copyright (c) 2023, Google Limited
  */
-#ifndef DRIVERS_CBMEM_CONSOLE_H
-#define DRIVERS_CBMEM_CONSOLE_H
+#ifndef __DRIVERS_CBMEM_CONSOLE_H
+#define __DRIVERS_CBMEM_CONSOLE_H
 
 #include <stdbool.h>
 
 bool cbmem_console_init_from_dt(void *fdt);
 
-#endif /* DRIVERS_CBMEM_CONSOLE_H */
+#endif /* __DRIVERS_CBMEM_CONSOLE_H */
 
diff --git a/core/include/drivers/gpio.h b/core/include/drivers/gpio.h
index 7ce00e1f9..b6406d45e 100644
--- a/core/include/drivers/gpio.h
+++ b/core/include/drivers/gpio.h
@@ -3,8 +3,8 @@
  * Copyright (c) 2016, Linaro Limited
  */
 
-#ifndef DRIVERS_GPIO_H
-#define DRIVERS_GPIO_H
+#ifndef __DRIVERS_GPIO_H
+#define __DRIVERS_GPIO_H
 
 #include <assert.h>
 #include <dt-bindings/gpio/gpio.h>
@@ -219,4 +219,4 @@ static inline TEE_Result gpio_register_provider(const void *fdt, int nodeoffset,
 					   data, DT_DRIVER_GPIO);
 }
 
-#endif	/* DRIVERS_GPIO_H */
+#endif	/* __DRIVERS_GPIO_H */
diff --git a/core/include/drivers/plic.h b/core/include/drivers/plic.h
index 148fc1e97..f1f260430 100644
--- a/core/include/drivers/plic.h
+++ b/core/include/drivers/plic.h
@@ -3,8 +3,8 @@
  * Copyright 2022-2023 NXP
  */
 
-#ifndef DRIVERS_PLIC_H
-#define DRIVERS_PLIC_H
+#ifndef __DRIVERS_PLIC_H
+#define __DRIVERS_PLIC_H
 
 #include <kernel/interrupt.h>
 #include <kernel/misc.h>
@@ -15,4 +15,4 @@ void plic_hart_init(void);
 void plic_it_handle(void);
 void plic_dump_state(void);
 
-#endif /*DRIVERS_PLIC_H*/
+#endif /*__DRIVERS_PLIC_H*/
diff --git a/core/include/drivers/regulator.h b/core/include/drivers/regulator.h
index e7c52a7b3..f020206d6 100644
--- a/core/include/drivers/regulator.h
+++ b/core/include/drivers/regulator.h
@@ -2,8 +2,8 @@
 /*
  * Copyright (c) 2023, STMicroelectronics
  */
-#ifndef DRIVERS_REGULATOR_H
-#define DRIVERS_REGULATOR_H
+#ifndef __DRIVERS_REGULATOR_H
+#define __DRIVERS_REGULATOR_H
 
 #include <assert.h>
 #include <bitstring.h>
@@ -315,4 +315,4 @@ static inline void regulator_get_range(struct regulator *regulator, int *min_uv,
  */
 TEE_Result regulator_supported_voltages(struct regulator *regulator,
 					struct regulator_voltages **voltages);
-#endif /* DRIVERS_REGULATOR_H */
+#endif /* __DRIVERS_REGULATOR_H */
diff --git a/core/include/drivers/rtc.h b/core/include/drivers/rtc.h
index c7f05be59..1e0df8203 100644
--- a/core/include/drivers/rtc.h
+++ b/core/include/drivers/rtc.h
@@ -3,8 +3,8 @@
  * Copyright 2022 Microchip.
  */
 
-#ifndef DRIVERS_RTC_H
-#define DRIVERS_RTC_H
+#ifndef __DRIVERS_RTC_H
+#define __DRIVERS_RTC_H
 
 #include <tee_api_types.h>
 #include <util.h>
@@ -127,4 +127,4 @@ static inline TEE_Result rtc_set_offset(long offset __unused)
 	return TEE_ERROR_NOT_SUPPORTED;
 }
 #endif
-#endif /* DRIVERS_RTC_H */
+#endif /* __DRIVERS_RTC_H */
diff --git a/core/include/drivers/stm32_gpio.h b/core/include/drivers/stm32_gpio.h
index adc85358c..706fb1bf4 100644
--- a/core/include/drivers/stm32_gpio.h
+++ b/core/include/drivers/stm32_gpio.h
@@ -3,8 +3,8 @@
  * Copyright (c) 2017-2023, STMicroelectronics
  */
 
-#ifndef DRIVERS_STM32_GPIO_H
-#define DRIVERS_STM32_GPIO_H
+#ifndef __DRIVERS_STM32_GPIO_H
+#define __DRIVERS_STM32_GPIO_H
 
 #include <assert.h>
 #include <drivers/pinctrl.h>
@@ -68,4 +68,4 @@ static inline void stm32_gpio_pinctrl_bank_pin(struct pinctrl_state *p __unused,
 {
 }
 #endif /*CFG_STM32_GPIO*/
-#endif /*DRIVERS_STM32_GPIO_H*/
+#endif /*__DRIVERS_STM32_GPIO_H*/
diff --git a/core/include/drivers/stm32_i2c.h b/core/include/drivers/stm32_i2c.h
index a1c83482c..0c955ade2 100644
--- a/core/include/drivers/stm32_i2c.h
+++ b/core/include/drivers/stm32_i2c.h
@@ -3,8 +3,8 @@
  * Copyright (c) 2017-2023, STMicroelectronics
  */
 
-#ifndef DRIVERS_STM32_I2C_H
-#define DRIVERS_STM32_I2C_H
+#ifndef __DRIVERS_STM32_I2C_H
+#define __DRIVERS_STM32_I2C_H
 
 #include <drivers/clk.h>
 #include <drivers/i2c.h>
@@ -278,4 +278,4 @@ static inline bool i2c_is_secure(struct i2c_handle_s *hi2c)
 	return hi2c->dt_status == DT_STATUS_OK_SEC;
 }
 
-#endif /* DRIVERS_STM32_I2C_H*/
+#endif /* __DRIVERS_STM32_I2C_H*/
diff --git a/core/include/drivers/stm32_iwdg.h b/core/include/drivers/stm32_iwdg.h
index f1ad73453..cab9c9c97 100644
--- a/core/include/drivers/stm32_iwdg.h
+++ b/core/include/drivers/stm32_iwdg.h
@@ -3,8 +3,8 @@
  * Copyright (c) 2018-2022, STMicroelectronics - All Rights Reserved
  */
 
-#ifndef DRIVERS_STM32_IWDG_H
-#define DRIVERS_STM32_IWDG_H
+#ifndef __DRIVERS_STM32_IWDG_H
+#define __DRIVERS_STM32_IWDG_H
 
 #include <stdbool.h>
 
@@ -30,4 +30,4 @@ TEE_Result stm32_get_iwdg_otp_config(paddr_t pbase,
 /* Refresh all registered IWDG watchdog instance */
 void stm32_iwdg_refresh(void);
 
-#endif /*DRIVERS_STM32_IWDG_H*/
+#endif /*__DRIVERS_STM32_IWDG_H*/
diff --git a/core/include/drivers/stm32_uart.h b/core/include/drivers/stm32_uart.h
index 250737f33..6054df72c 100644
--- a/core/include/drivers/stm32_uart.h
+++ b/core/include/drivers/stm32_uart.h
@@ -3,8 +3,8 @@
  * Copyright (c) 2017-2023, STMicroelectronics
  */
 
-#ifndef DRIVERS_STM32_UART_H
-#define DRIVERS_STM32_UART_H
+#ifndef __DRIVERS_STM32_UART_H
+#define __DRIVERS_STM32_UART_H
 
 #include <drivers/clk.h>
 #include <drivers/pinctrl.h>
@@ -42,4 +42,4 @@ void stm32_uart_init(struct stm32_uart_pdata *pd, vaddr_t base);
  */
 struct stm32_uart_pdata *stm32_uart_init_from_dt_node(void *fdt, int node);
 
-#endif /*DRIVERS_STM32_UART_H*/
+#endif /*__DRIVERS_STM32_UART_H*/
diff --git a/core/include/drivers/wdt.h b/core/include/drivers/wdt.h
index b07152e7b..8a22b5eb4 100644
--- a/core/include/drivers/wdt.h
+++ b/core/include/drivers/wdt.h
@@ -3,8 +3,8 @@
  * Copyright 2019 Broadcom.
  */
 
-#ifndef DRIVERS_WDT_H
-#define DRIVERS_WDT_H
+#ifndef __DRIVERS_WDT_H
+#define __DRIVERS_WDT_H
 
 #include <assert.h>
 #include <kernel/interrupt.h>
@@ -117,4 +117,4 @@ enum sm_handler_ret wdt_sm_handler(struct thread_smc_args *args __unused)
 }
 #endif
 
-#endif /* DRIVERS_WDT_H */
+#endif /* __DRIVERS_WDT_H */
diff --git a/core/include/gen-asm-defines.h b/core/include/gen-asm-defines.h
index ebf330f20..8b612a539 100644
--- a/core/include/gen-asm-defines.h
+++ b/core/include/gen-asm-defines.h
@@ -1,12 +1,12 @@
 /* SPDX-License-Identifier: BSD-2-Clause */
 /* Copyright (c) 2018, Linaro Limited */
 
-#ifndef GEN_ASM_DEFINES_H
-#define GEN_ASM_DEFINES_H
+#ifndef __GEN_ASM_DEFINES_H
+#define __GEN_ASM_DEFINES_H
 
 #define DEFINES void __defines(void); void __defines(void)
 
 #define DEFINE(def, val) \
 	asm volatile("\n.ascii \"==>" #def " %c0 " #val "\"" : : "i" (val));
 
-#endif /*GEN_ASM_DEFINES_H*/
+#endif /*__GEN_ASM_DEFINES_H*/
diff --git a/core/include/initcall.h b/core/include/initcall.h
index 28aa5ea75..db3d95e21 100644
--- a/core/include/initcall.h
+++ b/core/include/initcall.h
@@ -3,8 +3,8 @@
  * Copyright (c) 2014, Linaro Limited
  */
 
-#ifndef INITCALL_H
-#define INITCALL_H
+#ifndef __INITCALL_H
+#define __INITCALL_H
 
 #include <scattered_array.h>
 #include <tee_api_types.h>
diff --git a/core/include/io.h b/core/include/io.h
index 85365ce7a..85c885809 100644
--- a/core/include/io.h
+++ b/core/include/io.h
@@ -2,8 +2,8 @@
 /*
  * Copyright (c) 2014-2019, Linaro Limited
  */
-#ifndef IO_H
-#define IO_H
+#ifndef __IO_H
+#define __IO_H
 
 #include <compiler.h>
 #include <stdint.h>
@@ -300,4 +300,4 @@ static inline void io_clrsetbits8(vaddr_t addr, uint8_t clear_mask,
 		!(_cond); \
 	})
 
-#endif /*IO_H*/
+#endif /*__IO_H*/
diff --git a/core/include/keep.h b/core/include/keep.h
index 0dbe91005..622d1d65b 100644
--- a/core/include/keep.h
+++ b/core/include/keep.h
@@ -2,8 +2,8 @@
 /*
  * Copyright (c) 2015, Linaro Limited
  */
-#ifndef KEEP_H
-#define KEEP_H
+#ifndef __KEEP_H
+#define __KEEP_H
 
 #ifdef __ASSEMBLER__
 
@@ -45,4 +45,4 @@
 
 #endif /* __ASSEMBLER__ */
 
-#endif /*KEEP_H*/
+#endif /*__KEEP_H*/
diff --git a/core/include/kernel/abort.h b/core/include/kernel/abort.h
index ed0e5c0ca..4a0f5270d 100644
--- a/core/include/kernel/abort.h
+++ b/core/include/kernel/abort.h
@@ -3,8 +3,8 @@
  * Copyright (c) 2015-2021, Linaro Limited
  */
 
-#ifndef KERNEL_ABORT_H
-#define KERNEL_ABORT_H
+#ifndef __KERNEL_ABORT_H
+#define __KERNEL_ABORT_H
 
 #define ABORT_TYPE_UNDEF		0
 #define ABORT_TYPE_PREFETCH		1
@@ -40,5 +40,5 @@ bool abort_is_write_fault(struct abort_info *ai);
 void abort_print_current_ts(void);
 
 #endif /*__ASSEMBLER__*/
-#endif /*KERNEL_ABORT_H*/
+#endif /*__KERNEL_ABORT_H*/
 
diff --git a/core/include/kernel/dt.h b/core/include/kernel/dt.h
index 7a7bd9afd..c24d3bfa5 100644
--- a/core/include/kernel/dt.h
+++ b/core/include/kernel/dt.h
@@ -3,8 +3,8 @@
  * Copyright (c) 2016-2021, Linaro Limited
  */
 
-#ifndef KERNEL_DT_H
-#define KERNEL_DT_H
+#ifndef __KERNEL_DT_H
+#define __KERNEL_DT_H
 
 #include <compiler.h>
 #include <kernel/interrupt.h>
@@ -445,4 +445,4 @@ static inline int add_res_mem_dt_node(struct dt_descriptor *dt __unused,
 }
 
 #endif /* !CFG_DT */
-#endif /* KERNEL_DT_H */
+#endif /* __KERNEL_DT_H */
diff --git a/core/include/kernel/early_ta.h b/core/include/kernel/early_ta.h
index b7f9f38a6..8ad12e596 100644
--- a/core/include/kernel/early_ta.h
+++ b/core/include/kernel/early_ta.h
@@ -3,8 +3,8 @@
  * Copyright (c) 2017, 2020, Linaro Limited
  * Copyright (c) 2020, Arm Limited.
  */
-#ifndef KERNEL_EARLY_TA_H
-#define KERNEL_EARLY_TA_H
+#ifndef __KERNEL_EARLY_TA_H
+#define __KERNEL_EARLY_TA_H
 
 #include <kernel/embedded_ts.h>
 #include <scattered_array.h>
@@ -14,5 +14,5 @@
 #define for_each_early_ta(_ta) \
 	SCATTERED_ARRAY_FOREACH(_ta, early_tas, struct embedded_ts)
 
-#endif /* KERNEL_EARLY_TA_H */
+#endif /* __KERNEL_EARLY_TA_H */
 
diff --git a/core/include/kernel/embedded_ts.h b/core/include/kernel/embedded_ts.h
index d2ae758b5..aeaabffa2 100644
--- a/core/include/kernel/embedded_ts.h
+++ b/core/include/kernel/embedded_ts.h
@@ -3,8 +3,8 @@
  * Copyright (c) 2017, Linaro Limited
  * Copyright (c) 2020, Arm Limited.
  */
-#ifndef KERNEL_EMBEDDED_TS_H
-#define KERNEL_EMBEDDED_TS_H
+#ifndef __KERNEL_EMBEDDED_TS_H
+#define __KERNEL_EMBEDDED_TS_H
 
 #include <compiler.h>
 #include <kernel/linker.h>
@@ -33,5 +33,5 @@ TEE_Result emb_ts_open(const TEE_UUID *uuid,
 TEE_Result emb_ts_get_size(const struct ts_store_handle *h, size_t *size);
 TEE_Result emb_ts_get_tag(const struct ts_store_handle *h,
 			  uint8_t *tag, unsigned int *tag_len);
-#endif /* KERNEL_EMBEDDED_TS_H */
+#endif /* __KERNEL_EMBEDDED_TS_H */
 
diff --git a/core/include/kernel/handle.h b/core/include/kernel/handle.h
index eb4375dd3..9a7504d5c 100644
--- a/core/include/kernel/handle.h
+++ b/core/include/kernel/handle.h
@@ -3,8 +3,8 @@
  * Copyright (c) 2014, Linaro Limited
  * Copyright (c) 2020, Arm Limited
  */
-#ifndef KERNEL_HANDLE_H
-#define KERNEL_HANDLE_H
+#ifndef __KERNEL_HANDLE_H
+#define __KERNEL_HANDLE_H
 
 #include <stdbool.h>
 #include <stdint.h>
@@ -49,4 +49,4 @@ void *handle_put(struct handle_db *db, int handle);
  */
 void *handle_lookup(struct handle_db *db, int handle);
 
-#endif /*KERNEL_HANDLE_H*/
+#endif /*__KERNEL_HANDLE_H*/
diff --git a/core/include/kernel/ldelf_loader.h b/core/include/kernel/ldelf_loader.h
index 4d5e92c6f..fef590cd0 100644
--- a/core/include/kernel/ldelf_loader.h
+++ b/core/include/kernel/ldelf_loader.h
@@ -7,8 +7,8 @@
 #include <kernel/user_mode_ctx.h>
 #include <tee_api_types.h>
 
-#ifndef KERNEL_LDELF_LOADER_H
-#define KERNEL_LDELF_LOADER_H
+#ifndef __KERNEL_LDELF_LOADER_H
+#define __KERNEL_LDELF_LOADER_H
 
 TEE_Result ldelf_load_ldelf(struct user_mode_ctx *uctx);
 TEE_Result ldelf_init_with_ldelf(struct ts_session *sess,
@@ -21,4 +21,4 @@ TEE_Result ldelf_dlopen(struct user_mode_ctx *uctx, TEE_UUID *uuid,
 TEE_Result ldelf_dlsym(struct user_mode_ctx *uctx, TEE_UUID *uuid,
 		       const char *sym, size_t symlen, vaddr_t *val);
 
-#endif /* KERNEL_LDELF_LOADER_H */
+#endif /* __KERNEL_LDELF_LOADER_H */
diff --git a/core/include/kernel/ldelf_syscalls.h b/core/include/kernel/ldelf_syscalls.h
index 5dd89a7f2..62b410ad7 100644
--- a/core/include/kernel/ldelf_syscalls.h
+++ b/core/include/kernel/ldelf_syscalls.h
@@ -11,8 +11,8 @@
 #include <tee_api_types.h>
 #include <types_ext.h>
 
-#ifndef KERNEL_LDELF_SYSCALLS_H
-#define KERNEL_LDELF_SYSCALLS_H
+#ifndef __KERNEL_LDELF_SYSCALLS_H
+#define __KERNEL_LDELF_SYSCALLS_H
 
 struct system_ctx {
 	struct handle_db db;
@@ -39,4 +39,4 @@ TEE_Result ldelf_syscall_remap(unsigned long old_va, vaddr_t *new_va,
 TEE_Result ldelf_syscall_gen_rnd_num(void *buf, size_t num_bytes);
 void ldelf_sess_cleanup(struct ts_session *sess);
 
-#endif /* KERNEL_LDELF_SYSCALLS_H */
+#endif /* __KERNEL_LDELF_SYSCALLS_H */
diff --git a/core/include/kernel/misc.h b/core/include/kernel/misc.h
index 35cfa5f13..d204a3ae7 100644
--- a/core/include/kernel/misc.h
+++ b/core/include/kernel/misc.h
@@ -3,8 +3,8 @@
  * Copyright (c) 2014, STMicroelectronics International N.V.
  */
 
-#ifndef KERNEL_MISC_H
-#define KERNEL_MISC_H
+#ifndef __KERNEL_MISC_H
+#define __KERNEL_MISC_H
 
 #include <assert.h>
 #include <kernel/misc_arch.h>
@@ -23,4 +23,4 @@ static inline size_t __noprof get_core_pos(void)
 	return __get_core_pos();
 }
 
-#endif /*KERNEL_MISC_H*/
+#endif /*__KERNEL_MISC_H*/
diff --git a/core/include/kernel/msg_param.h b/core/include/kernel/msg_param.h
index 841c3bfa4..2cf3da657 100644
--- a/core/include/kernel/msg_param.h
+++ b/core/include/kernel/msg_param.h
@@ -26,8 +26,8 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
-#ifndef KERNEL_MSG_PARAM_H
-#define KERNEL_MSG_PARAM_H
+#ifndef __KERNEL_MSG_PARAM_H
+#define __KERNEL_MSG_PARAM_H
 
 #include <compiler.h>
 #include <kernel/msg_param.h>
@@ -81,4 +81,4 @@ static inline bool msg_param_attr_is_tmem(uint64_t attr)
 	}
 }
 
-#endif	/*KERNEL_MSG_PARAM_H*/
+#endif	/*__KERNEL_MSG_PARAM_H*/
diff --git a/core/include/kernel/mutex.h b/core/include/kernel/mutex.h
index e4c470b0f..c701188d7 100644
--- a/core/include/kernel/mutex.h
+++ b/core/include/kernel/mutex.h
@@ -2,8 +2,8 @@
 /*
  * Copyright (c) 2014-2017, Linaro Limited
  */
-#ifndef KERNEL_MUTEX_H
-#define KERNEL_MUTEX_H
+#ifndef __KERNEL_MUTEX_H
+#define __KERNEL_MUTEX_H
 
 #include <kernel/refcount.h>
 #include <kernel/wait_queue.h>
@@ -101,5 +101,5 @@ void condvar_broadcast(struct condvar *cv);
 void condvar_wait(struct condvar *cv, struct mutex *m);
 #endif
 
-#endif /*KERNEL_MUTEX_H*/
+#endif /*__KERNEL_MUTEX_H*/
 
diff --git a/core/include/kernel/panic.h b/core/include/kernel/panic.h
index 9dc38dce1..3d4fa55b2 100644
--- a/core/include/kernel/panic.h
+++ b/core/include/kernel/panic.h
@@ -3,8 +3,8 @@
  * Copyright (c) 2014, STMicroelectronics International N.V.
  */
 
-#ifndef KERNEL_PANIC_H
-#define KERNEL_PANIC_H
+#ifndef __KERNEL_PANIC_H
+#define __KERNEL_PANIC_H
 
 #include <compiler.h>
 
@@ -39,4 +39,4 @@ void __do_panic(const char *file, const int line, const char *func,
  */
 void cpu_idle(void);
 
-#endif /*KERNEL_PANIC_H*/
+#endif /*__KERNEL_PANIC_H*/
diff --git a/core/include/kernel/pseudo_ta.h b/core/include/kernel/pseudo_ta.h
index 132c59f11..965ca0d43 100644
--- a/core/include/kernel/pseudo_ta.h
+++ b/core/include/kernel/pseudo_ta.h
@@ -2,8 +2,8 @@
 /*
  * Copyright (c) 2015, Linaro Limited
  */
-#ifndef KERNEL_PSEUDO_TA_H
-#define KERNEL_PSEUDO_TA_H
+#ifndef __KERNEL_PSEUDO_TA_H
+#define __KERNEL_PSEUDO_TA_H
 
 #include <assert.h>
 #include <compiler.h>
@@ -95,5 +95,5 @@ TEE_Result from_bounce_params(uint32_t param_types,
 			      TEE_Param bparams[TEE_NUM_PARAMS],
 			      TEE_Param *eparams);
 
-#endif /* KERNEL_PSEUDO_TA_H */
+#endif /* __KERNEL_PSEUDO_TA_H */
 
diff --git a/core/include/kernel/spinlock.h b/core/include/kernel/spinlock.h
index fc3e2e474..e7a84d32b 100644
--- a/core/include/kernel/spinlock.h
+++ b/core/include/kernel/spinlock.h
@@ -4,8 +4,8 @@
  * Copyright (c) 2016, Linaro Limited
  */
 
-#ifndef KERNEL_SPINLOCK_H
-#define KERNEL_SPINLOCK_H
+#ifndef __KERNEL_SPINLOCK_H
+#define __KERNEL_SPINLOCK_H
 
 #define SPINLOCK_LOCK       1
 #define SPINLOCK_UNLOCK     0
@@ -129,4 +129,4 @@ static inline void cpu_spin_unlock_xrestore(unsigned int *lock,
 }
 #endif /* __ASSEMBLER__ */
 
-#endif /* KERNEL_SPINLOCK_H */
+#endif /* __KERNEL_SPINLOCK_H */
diff --git a/core/include/kernel/thread.h b/core/include/kernel/thread.h
index 022aeec58..b7f497e10 100644
--- a/core/include/kernel/thread.h
+++ b/core/include/kernel/thread.h
@@ -5,8 +5,8 @@
  * Copyright (c) 2020-2021, Arm Limited
  */
 
-#ifndef KERNEL_THREAD_H
-#define KERNEL_THREAD_H
+#ifndef __KERNEL_THREAD_H
+#define __KERNEL_THREAD_H
 
 #ifndef __ASSEMBLER__
 #include <types_ext.h>
@@ -396,4 +396,4 @@ void *thread_rpc_shm_cache_alloc(enum thread_shm_cache_user user,
 
 #endif /*__ASSEMBLER__*/
 
-#endif /*KERNEL_THREAD_H*/
+#endif /*__KERNEL_THREAD_H*/
diff --git a/core/include/kernel/unwind.h b/core/include/kernel/unwind.h
index 9a01b4613..8ecf34359 100644
--- a/core/include/kernel/unwind.h
+++ b/core/include/kernel/unwind.h
@@ -4,8 +4,8 @@
  * Copyright (c) 2020, Huawei Technologies Co., Ltd
  */
 
-#ifndef KERNEL_UNWIND
-#define KERNEL_UNWIND
+#ifndef __KERNEL_UNWIND
+#define __KERNEL_UNWIND
 
 #include <types_ext.h>
 
@@ -27,4 +27,4 @@ static inline void *unw_get_kernel_stack(void)
 }
 #endif /* CFG_UNWIND  */
 
-#endif /*KERNEL_UNWIND*/
+#endif /*__KERNEL_UNWIND*/
diff --git a/core/include/kernel/user_ta.h b/core/include/kernel/user_ta.h
index 942c948f6..667122b5c 100644
--- a/core/include/kernel/user_ta.h
+++ b/core/include/kernel/user_ta.h
@@ -3,8 +3,8 @@
  * Copyright (c) 2015, Linaro Limited
  * Copyright (c) 2020, Arm Limited
  */
-#ifndef KERNEL_USER_TA_H
-#define KERNEL_USER_TA_H
+#ifndef __KERNEL_USER_TA_H
+#define __KERNEL_USER_TA_H
 
 #include <assert.h>
 #include <kernel/tee_ta_manager.h>
@@ -70,4 +70,4 @@ static inline TEE_Result tee_ta_init_user_ta_session(
 #endif
 
 
-#endif /*KERNEL_USER_TA_H*/
+#endif /*__KERNEL_USER_TA_H*/
diff --git a/core/include/kernel/virtualization.h b/core/include/kernel/virtualization.h
index c4a7f69fe..f14904e56 100644
--- a/core/include/kernel/virtualization.h
+++ b/core/include/kernel/virtualization.h
@@ -1,8 +1,8 @@
 /* SPDX-License-Identifier: BSD-2-Clause */
 /* Copyright (c) 2018, EPAM Systems. All rights reserved. */
 
-#ifndef KERNEL_VIRTUALIZATION_H
-#define KERNEL_VIRTUALIZATION_H
+#ifndef __KERNEL_VIRTUALIZATION_H
+#define __KERNEL_VIRTUALIZATION_H
 
 #include <mm/core_mmu.h>
 #include <stdbool.h>
@@ -135,4 +135,4 @@ static inline uint16_t virt_find_guest_by_cookie(uint64_t cookie __unused)
 { return 0; }
 #endif
 
-#endif	/* KERNEL_VIRTUALIZATION_H */
+#endif	/* __KERNEL_VIRTUALIZATION_H */
diff --git a/core/include/kernel/wait_queue.h b/core/include/kernel/wait_queue.h
index a13c02b6c..20691e7fe 100644
--- a/core/include/kernel/wait_queue.h
+++ b/core/include/kernel/wait_queue.h
@@ -2,8 +2,8 @@
 /*
  * Copyright (c) 2015, Linaro Limited
  */
-#ifndef KERNEL_WAIT_QUEUE_H
-#define KERNEL_WAIT_QUEUE_H
+#ifndef __KERNEL_WAIT_QUEUE_H
+#define __KERNEL_WAIT_QUEUE_H
 
 #include <types_ext.h>
 #include <sys/queue.h>
@@ -60,5 +60,5 @@ void wq_promote_condvar(struct wait_queue *wq, struct condvar *cv,
 			int lineno);
 bool wq_have_condvar(struct wait_queue *wq, struct condvar *cv);
 
-#endif /*KERNEL_WAIT_QUEUE_H*/
+#endif /*__KERNEL_WAIT_QUEUE_H*/
 
diff --git a/core/include/mm/pgt_cache.h b/core/include/mm/pgt_cache.h
index 13cd9b210..4f3e37e26 100644
--- a/core/include/mm/pgt_cache.h
+++ b/core/include/mm/pgt_cache.h
@@ -2,8 +2,8 @@
 /*
  * Copyright (c) 2016, Linaro Limited
  */
-#ifndef MM_PGT_CACHE_H
-#define MM_PGT_CACHE_H
+#ifndef __MM_PGT_CACHE_H
+#define __MM_PGT_CACHE_H
 
 #ifdef CFG_WITH_LPAE
 #define PGT_SIZE	(4 * 1024)
@@ -136,4 +136,4 @@ static inline void pgt_set_used_entries(struct pgt *pgt __unused,
 
 #endif
 
-#endif /*MM_PGT_CACHE_H*/
+#endif /*__MM_PGT_CACHE_H*/
diff --git a/core/include/mm/tee_pager.h b/core/include/mm/tee_pager.h
index 4e42e64eb..3c5b63017 100644
--- a/core/include/mm/tee_pager.h
+++ b/core/include/mm/tee_pager.h
@@ -4,8 +4,8 @@
  * Copyright (c) 2014, STMicroelectronics International N.V.
  */
 
-#ifndef MM_TEE_PAGER_H
-#define MM_TEE_PAGER_H
+#ifndef __MM_TEE_PAGER_H
+#define __MM_TEE_PAGER_H
 
 #include <kernel/abort.h>
 #include <kernel/panic.h>
diff --git a/core/include/optee_msg.h b/core/include/optee_msg.h
index 7cc795d74..5fa626b0e 100644
--- a/core/include/optee_msg.h
+++ b/core/include/optee_msg.h
@@ -2,8 +2,8 @@
 /*
  * Copyright (c) 2015-2020, Linaro Limited
  */
-#ifndef _OPTEE_MSG_H
-#define _OPTEE_MSG_H
+#ifndef __OPTEE_MSG_H
+#define __OPTEE_MSG_H
 
 #include <compiler.h>
 #include <types_ext.h>
@@ -342,4 +342,4 @@ struct optee_msg_arg {
 #define OPTEE_MSG_CMD_STOP_ASYNC_NOTIF	U(7)
 #define OPTEE_MSG_FUNCID_CALL_WITH_ARG	U(0x0004)
 
-#endif /* _OPTEE_MSG_H */
+#endif /* __OPTEE_MSG_H */
diff --git a/core/include/signed_hdr.h b/core/include/signed_hdr.h
index edeb446cc..0aacd8fee 100644
--- a/core/include/signed_hdr.h
+++ b/core/include/signed_hdr.h
@@ -2,8 +2,8 @@
 /*
  * Copyright (c) 2015, Linaro Limited
  */
-#ifndef SIGNED_HDR_H
-#define SIGNED_HDR_H
+#ifndef __SIGNED_HDR_H
+#define __SIGNED_HDR_H
 
 #include <inttypes.h>
 #include <stdlib.h>
@@ -225,4 +225,4 @@ TEE_Result shdr_verify_signature2(struct shdr_pub_key *key,
  */
 TEE_Result shdr_verify_signature(const struct shdr *shdr);
 
-#endif /*SIGNED_HDR_H*/
+#endif /*__SIGNED_HDR_H*/
diff --git a/core/include/tee/cache.h b/core/include/tee/cache.h
index 568b2515e..cb412104e 100644
--- a/core/include/tee/cache.h
+++ b/core/include/tee/cache.h
@@ -2,11 +2,11 @@
 /*
  * Copyright (c) 2015, Linaro Limited
  */
-#ifndef TEE_CACHE_H
-#define TEE_CACHE_H
+#ifndef __TEE_CACHE_H
+#define __TEE_CACHE_H
 
 #include <utee_types.h>
 
 TEE_Result cache_operation(enum utee_cache_operation op, void *va, size_t len);
 
-#endif /* TEE_CACHE_H */
+#endif /* __TEE_CACHE_H */
diff --git a/core/include/tee/entry_std.h b/core/include/tee/entry_std.h
index d228badeb..d9ddb6200 100644
--- a/core/include/tee/entry_std.h
+++ b/core/include/tee/entry_std.h
@@ -4,8 +4,8 @@
  * Copyright (c) 2014, STMicroelectronics International N.V.
  */
 
-#ifndef TEE_ENTRY_STD_H
-#define TEE_ENTRY_STD_H
+#ifndef __TEE_ENTRY_STD_H
+#define __TEE_ENTRY_STD_H
 
 #include <kernel/thread.h>
 #include <optee_msg.h>
@@ -23,4 +23,4 @@ TEE_Result __tee_entry_std(struct optee_msg_arg *arg, uint32_t num_params);
 /* Get list head for sessions opened from non-secure */
 void nsec_sessions_list_head(struct tee_ta_session_head **open_sessions);
 
-#endif /* TEE_ENTRY_STD_H */
+#endif /* __TEE_ENTRY_STD_H */
-- 
2.47.2

